include/glow/Backends/DeviceManager.h

```diff
@@ -89,6 +89,18 @@ class DeviceManager {
   /// \returns the name of backend that powers this Device.
   llvm::StringRef getBackendName() { return config_.backendName; }
 
+  /// \returns a string contains the node name (e.g Add, Sub) that won't be
+  /// supported by this DeviceManger. It doesn't mean that those nodes kind are
+  /// not supported by this device. It just means user don't want these kind of
+  /// nodes running on this device.
+  llvm::StringRef getNonSupprtedNodes() const {
```

Comment:
It is probably better to implement `getParamByName(llvm::StringRef name)` to make it more reusable. And then you can just call it like `getParamByName("nonSupportedOp")`